caiman/motion_correction.py

```diff
@@ -1552,13 +1594,50 @@ def register_translation(src_image, target_image, upsample_factor=1,
         raise NotImplementedError("Error: register_translation only supports "
                                   "subpixel registration for 2D images")
 
+    if HAS_CUDA and use_cuda:
+        from skcuda.fft import Plan
```

Comment:
I think we might want an alternate environment file that has conda install this as a dependency

Reply:
I would not do it for the moment, but I would add a comment in the README alerting the users about it.

Reply:
Done.